build.gradle

```diff
@@ -154,8 +154,8 @@ tasks.withType(ValidatePlugins).configureEach {
 }
 
 signing {
-    // Require publications to be signed on CI. Otherwise, publication will be signed only if keys are provided.
-    required isCI
+    // Require publications to be signed on CI and publishing task included in the TaskGraph
+    required { isCI && gradle.taskGraph.hasTask(":publishPlugins") }
```

Comment:
IMO we can remove `isCI`. Regardless of from where the plugin is being published, signing should be required.

Reply:
publishing locally the plugin shouldn't require signing. Good news is we are not using the `publishPlugins` for local  builds and `publishToMavenLocal` doesn't include the task required to sign. 

done https://github.com/gradle/android-cache-fix-gradle-plugin/pull/479/commits/f15a48bdbedfcfca982f65719f5e72d6ba5c0130